callback_plugins/ansible_version_check.py

```diff
@@ -0,0 +1,33 @@
+# -*- encoding:utf-8 -*-
+# https://adamj.eu/tech/2016/08/19/restricting-the-ansible-version-in-use/
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import sys
+
+import ansible
+
+try:
+    # Version 2.0+
+    from ansible.plugins.callback import CallbackBase
+except ImportError:
+    CallbackBase = object
+
+
+def print_red_bold(text):
```

Comment:
There's a built-in with the callback system for printing 'fail' type text, using 'print' directly should be avoided for this. I'll come back with a proper code example in a few minutes.